src/phoenix/config.py

```diff
@@ -219,4 +227,10 @@ def get_env_enable_prometheus() -> bool:
     )
 
 
+def get_env_client_headers() -> Optional[Mapping[str, str]]:
```

Comment:
```suggestion
def get_env_client_headers() -> Optional[Dict[str, str]]:
```

Suggestion: Make the return type as specific as possible.